pex/pex.py

```diff
@@ -620,6 +620,9 @@ def execute_interpreter(self):
             else:
                 args = args[index:]
                 break
+        else:
+            # All the args were python options
```

Comment:
The comment helps, and this is just me talking to the void, but I have to do this to re-prove to myself how for / else works everytime - such odd syntax:
```python
>>> def loop(brk):
...     for x in range(5):
...         if brk and x == 3:
...             break
...         print(x)
...     else:
...         print("else")
...
>>> loop(False)
0
1
2
3
4
else
>>> loop(True)
0
1
2
>>>
```